Make it possible to trigger XLA/slow tests via commit message.

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#18345 Make it possible to trigger XLA/slow tests via commit message.**

Four variants are supported: `[xla ci] [ci xla] [xla test] [test xla]`; substitute
xla with slow for slow tests.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D14584557](https://our.internmc.facebook.com/intern/diff/D14584557)

ezyang committed:
Make it possible to trigger XLA/slow tests via commit message.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

Make it possible to trigger XLA/slow tests via commit message.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

gh-metadata: pytorch pytorch 18345 gh/ezyang/40/head

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

Make it possible to trigger XLA/slow tests via commit message.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

gh-metadata: pytorch pytorch 18345 gh/ezyang/40/head

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

Make it possible to trigger XLA/slow tests via commit message.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

gh-metadata: pytorch pytorch 18345 gh/ezyang/40/head

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

Make it possible to trigger XLA/slow tests via commit message.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

gh-metadata: pytorch pytorch 18345 gh/ezyang/40/head

ailzhang reviewed (approved):
This is super useful!! Thanks @ezyang !

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

Make it possible to trigger XLA/slow tests via commit message.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

gh-metadata: pytorch pytorch 18345 gh/ezyang/40/head

ezyang committed:
Update on "Make it possible to trigger XLA/slow tests via commit message."

Make it possible to trigger XLA/slow tests via commit message.

[test xla]

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

gh-metadata: pytorch pytorch 18345 gh/ezyang/40/head
